.github/workflows/ci-tests-fabric.yml

```diff
@@ -138,7 +138,7 @@ jobs:
       # NOTE: do not include coverage report here, see: https://github.com/nedbat/coveragepy/issues/1003
       run: |
         python -m coverage run --source ${{ env.COVERAGE_SCOPE }} \
-          -m pytest -v --timeout=30 --durations=50
+          -m pytest -v --timeout=30 --durations=50  --random-order-seed=42
```

Comment:
Shouldn't we use a random seed every time instead of a fixed one?

Otherwise there might still be interactions

Reply:
@awaelchli had a good point with reproducible locally, so we can make see for a day. Would it work for you?

Reply:
I would assume that the seed used gets printed. Either way it's fine with me

Reply:
ok, that would be also an option :)